lib/iris/fileformats/netcdf/saver.py

```diff
@@ -48,28 +46,17 @@
 import iris.io
 import iris.util
 
-# Show actions activation statistics.
-DEBUG = False
+# Configure the logger : shared logger for all in 'iris.fileformats.netcdf'.
+from .. import __name__ as _parent_module_name
+
+logger = iris.config.get_logger(_parent_module_name)
+del _parent_module_name
 
-# Configure the logger.
-logger = iris.config.get_logger(__name__)
 
 # Standard CML spatio-temporal axis names.
 SPATIO_TEMPORAL_AXES = ["t", "z", "y", "x"]
 
-# Pass through CF attributes:
-#  - comment
-#  - Conventions
-#  - flag_masks
-#  - flag_meanings
-#  - flag_values
-#  - history
-#  - institution
-#  - reference
-#  - source
-#  - title
-#  - positive
-#
```

Comment:
Why have these been removed?

Reply:
I think **_this_** is a legacy comment which is probably now out-of-date.
Exactly what "Pass through" was supposed to mean here isn't clear to me, but 
  * some of these (like the flag_ ones) are in the next-but-one list [`_CF_DATA_ATTRS`](https://github.com/SciTools/iris/blob/56a97d66b8404f96ef61fe2d52433eb58373f338/lib/iris/fileformats/netcdf.py#L99)
  * others (e.g. history/conventions) are now in [`_CF_GLOBAL_ATTRS`](https://github.com/SciTools/iris/blob/56a97d66b8404f96ef61fe2d52433eb58373f338/lib/iris/fileformats/netcdf.py#L110)
  * while others again (e.g. reference/source/institution) don't seem to be specially handled _anywhere_ 
  
So I reckon this list is simply obsolete.

Reply:
Thanks for the explanation